src/Devices/Artemis.Plugins.Devices.Wooting/WootingModule.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Artemis.Core.Modules;
+using Artemis.Plugins.Devices.Wooting.Services;
+using RGB.NET.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Artemis.Plugins.Devices.Wooting;
+
+public class WootingModule : Module<WootingDataModel>
+{
+    private readonly WootingAnalogService _analogService;
+    private readonly WootingProfileService _profileService;
+
+    public override List<IModuleActivationRequirement> ActivationRequirements { get; } = new();
+
+    public WootingModule(WootingAnalogService service, WootingProfileService profileService)
+    {
+        _analogService = service;
+        _profileService = profileService;
+    }
+
+    public override void Enable()
+    {
+        foreach (var item in _analogService.Devices)
+        {
+            DataModel.AddDynamicChild(item.Info.device_name, new WootingDeviceDataModel());
+        }
```

Comment:
```suggestion
        foreach (WootingAnalogDevice item in _analogService.Devices)
            DataModel.AddDynamicChild(item.Info.device_name, new WootingDeviceDataModel());
```

Var usage and single-line bodies can be without brackets

Reply:
Oh var usage in the rest of the file too ofc